### PR TITLE
Rhodes NFT QN - fixed bids not captured

### LINKED ISSUE #3312: NFT query node - Bids not captured
During testing of NFTs we've noticed an auction that has both `bids: []` and `lastBid: null`, even though we placed a couple of bids for it.

You can see a block where one of the bids was made [here](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F54.172.37.177.nip.io%2Fws-rpc#/explorer/query/0xbd3d21113a315b7956dd6b6c1d8d348532f64e9f3f5579db09a50f1f5bbf86dd) (you will need to set types in the settings for this to work)

You can query the NFT details [here](https://54.172.37.177.nip.io/orion/graphql):

```graphql
{
  ownedNfts(where: { video: { id_eq: "6" } }) {
    id
    creatorRoyalty

    transactionalStatus {
      __typename
      ... on TransactionalStatusAuction {
        auction {
          startsAtBlock
          isCompleted

          endedAtBlock
          plannedEndAtBlock
          minimalBidStep
          startingPrice
          auctionType {
            __typename
            ... on AuctionTypeOpen {
              bidLockingTime
            }
            ... on AuctionTypeEnglish {
              extensionPeriod
              duration
            }
          }
          bids {
            id
            bidder {
              id
              handle
            }
            amount
            createdInBlock
          }
          lastBid {
            id
            isCanceled
          }
        }
      }
    }
  }
}
```

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201899819496282/1201899832929440) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by zeeshanakram3**
fixes #3312

### CODE REVIEW

File: tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import BN from 'bn.js'
+import { Api } from '../../../Api'
+import { BaseQueryNodeFixture, FixtureRunner } from '../../../Fixture'
+import { QueryNodeApi } from '../../../QueryNodeApi'
+import { Utils } from '../../../utils'
+import { IMember } from '../createMembers'
+import { PlaceBidsInAuctionFixture } from './placeBidsInAuction'
+import { assertNftOwner, assertAuctionAndBids } from './utils'
+
+export class NftEnglishAuctionWithExtensionFixture extends BaseQueryNodeFixture {
```
Comment (ondratra) [inline tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts]: This file is not actually used in this PR, or is it? I think you mistakenly commited this file that is originally introduced https://github.com/Joystream/joystream/pull/3408 . If that's the case you can remove it from this commit. It is causing compilation error anyway (see https://github.com/Joystream/joystream/runs/5553666098?check_suite_focus=true#step:4:334)

**Review by ondratra — CHANGES_REQUESTED** [review]
PR looks good; there is just one compilation issue in CI.

File: tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts (hunk 1 above)
Comment (zeeshanakram3) [reply tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts]: You are right; this file is part of #3408. However, `assertAuctionAndBids` [check](https://github.com/Joystream/joystream/blob/b957cf68d0f1c66c6f0be2f9cba9989b8ba9eae2/tests/network-tests/src/fixtures/content/nft/englishAuctionWithExtension.ts#L75 ) was introduced for the NFT fixtures in this PR, So I added this file (mistakenly) again in the PR. I am removing this file, and once the PRs are merged, I will create another PR for adding the `assertAuctionAndBids` check in this fixture.

**Review by ondratra — APPROVED** [review]
LGTM

**Comment by kdembler** [comment]
@zeeshanakram3  Seems some Prettier checks are failing here?

**Comment by ondratra** [comment]
Auction is breaking the tests and checks at the moment, but we can fix lint later on together with auction changes.

**Comment by zeeshanakram3** [comment]
@kdembler @ondratra I have fixed the Prettier checks failing issue.

**Review by kdembler — APPROVED** [review]

**Pull request merged**